src/ui/Api.js

```diff
@@ -11,12 +11,12 @@ export default class Api {
       // here we assume we want to create a new deck
       const resp = await fetch("/api/v2/deck/new", { method: "POST" });
       const response = await resp.json();
-      return response;
+      return "response";
```

Comment:
I think some of these changes are also in your last PR, so I'll not comment on those here.